Multiplatform/Navigation/Regular/SidebarView.swift

```diff
@@ -48,6 +56,16 @@ struct SidebarView: View {
         }
         .modifier(RegularNowPlayingBarModifier())
         .environment(\.libraryDataProvider, selection?.provider.libraryProvider ?? MockLibraryDataProvider())
+        .onReceive(NotificationCenter.default.publisher(for: Navigation.navigateArtistNotification)) { notification in
+            if let id = notification.object as? String {
+                navigationPath.append(Navigation.ArtistLoadDestination(artistId: id))
+            }
+        }
+        .onReceive(NotificationCenter.default.publisher(for: Navigation.navigateAlbumNotification)) { notification in
+            if let id = notification.object as? String {
+                navigationPath.append(Navigation.AlbumLoadDestination(albumId: id))
```

Comment:
This needs to check if the album is downloaded and change the data provider. This should work similar to the way playlists are navigated to, see https://github.com/rasmuslos/AmpFin/blob/1cb796796e873112bcf73b8ed190698cb3ed8adb/Multiplatform/Navigation/Regular/Sidebar%2BEntries.swift#L34